gosuper/gosuper/api.go

```diff
@@ -236,6 +237,15 @@ func LogToDisplayControl(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if loaded, err := systemd.Dbus.GetUnitProperty(serviceName, "LoadState"); err != nil {
+		sendError(fmt.Errorf("Unable to get log to display load status: %v", err))
+		return
+	} else if loaded.Value.String() == `"not-found"` {
```

Comment:
Is there maybe an exit code that can be checked for, rather than a hardcoded string?